x-pack/plugins/uptime/server/lib/requests/get_network_events.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UMElasticsearchQueryFn } from '../adapters/framework';
+import { NetworkEvent } from '../../../common/runtime_types';
+
+interface GetNetworkEventsParams {
+  checkGroup: string;
+  stepIndex: string;
+}
+
+export const getNetworkEvents: UMElasticsearchQueryFn<
+  GetNetworkEventsParams,
+  NetworkEvent[]
+> = async ({ uptimeEsClient, checkGroup, stepIndex }) => {
+  const params = {
+    query: {
+      bool: {
+        filter: [
+          { term: { 'synthetics.type': 'journey/network_info' } },
+          { term: { 'monitor.check_group': checkGroup } },
+          { term: { 'synthetics.step.index': Number(stepIndex) } },
+        ],
+      },
+    },
+    size: 1000,
```

Comment:
Can we please document this limit by a comment here?

Reply:
👍